### ISSUE TITLE
Error while running python training_validation.py file

### ISSUE THREAD

**Issue by Preetham-Vijay** [neuhanli/YJY#1]
c:\Users\preet\Desktop\FYP\NHGNN\.conda\lib\site-packages\torchvision\models\_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
c:\Users\preet\Desktop\FYP\NHGNN\.conda\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=MobileNet_V3_Small_Weights.IMAGENET1K_V1`. You can also use `weights=MobileNet_V3_Small_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
cuda_name: cuda:0
Learning rate:  0.0005
Epochs:  1000, can you let me know how to fix this

**Comment by neuhanli** [comment]
Just follow the steps in the README to run training.py; you can ignore training_validation.py, as this is just my intermediate experimental program.
        I will delete the training_validation.py.
    
         
        
            
                韩立

            
        
     
    
  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ***@***.***>
            
        
      
      
        
         Date 
        
        
        11/3/2024 16:48
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              [neuhanli/ImageDTA] Error while running python training_validation.py file (Issue #1)
        
      
    
  

c:\Users\preet\Desktop\FYP\NHGNN.conda\lib\site-packages\torchvision\models_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
warnings.warn(
c:\Users\preet\Desktop\FYP\NHGNN.conda\lib\site-packages\torchvision\models_utils.py:223: UserWarning: Arguments other than a weight enum or None for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing weights=MobileNet_V3_Small_Weights.IMAGENET1K_V1. You can also use weights=MobileNet_V3_Small_Weights.DEFAULT to get the most up-to-date weights.
warnings.warn(msg)
cuda_name: cuda:0
Learning rate:  0.0005
Epochs:  1000, can you let me know how to fix this

**Comment by Preetham-Vijay** [comment]
Thank you
